pkg/engine/imageVerify.go

```diff
@@ -25,9 +25,43 @@ import (
 	"k8s.io/apimachinery/pkg/apis/meta/v1/unstructured"
 )
 
+func getMatchingImages(images map[string]map[string]apiutils.ImageInfo, rule *kyvernov1.Rule) []apiutils.ImageInfo {
+	imageInfos := []apiutils.ImageInfo{}
+	for _, infoMap := range images {
+		for _, imageInfo := range infoMap {
+			image := imageInfo.String()
+			for _, verifyImage := range rule.VerifyImages {
+				verifyImage = *verifyImage.Convert()
+				if imageMatches(image, verifyImage.ImageReferences) {
+					imageInfos = append(imageInfos, imageInfo)
+				}
+			}
+		}
+	}
+	return imageInfos
+}
+
+func requireValidation(policyContext *PolicyContext, rule *kyvernov1.Rule) ([]apiutils.ImageInfo, error) {
```

Comment:
Since this method has changed, can we please rename it to match what it does? For example, `extractMatchingImages`?

Reply:
fixed